src/actions/loginActions.js

```diff
@@ -0,0 +1,20 @@
+import {
+  LOGIN_REQUEST,
+  LOGIN_SUCCESS,
+  LOGIN_FAILURE,
+} from '../actionTypes/loginActionType';
+
+export const loginRequestAction = payload => ({
+  type: LOGIN_REQUEST,
+  payload,
+});
+
+export const loginSuccessAction = response => ({
+  type: LOGIN_SUCCESS,
+  token: response.token,
```

Comment:
We'll need the username of the user as well.

Reply:
Yeah, @TheoOkafor. I Implemented a function to handle that.